src/record.ts

```diff
@@ -140,10 +141,14 @@ export default class BamRecord {
   }
 
   qual() {
-    return this.qualRaw()?.join(' ')
+    const qual = this.qualRaw()
+    return qual?.join(' ') || ''
```

Comment:
doesn't this return empty string if the qualRaw is undefined? wouldn't we want to return undefined instead?

Reply:
ya I see it returned undefined before, changed back